### DISCUSSION TITLE
Core Conversion

### DISCUSSION THREAD

**Discussion by DanCadwellcs** [mono/taglib-sharp#294] [Q&A] [answered]
I have just finished moving this solution to net6.  Is there any interest in me uploading this solution?  VS 2022 and Core 6.

**Comment by knuxbbs** [comment, marked as answer]
Well, it's nice to hear that. Why not open a pull request?

**Reply by DanCadwellcs** [reply]
I pushed up fork.  This is my first.  Please let me know if I did something wrong.

Dan

**Reply by knuxbbs** [reply]
I don't see anything in your fork. Are you working in this branch? https://github.com/Cadwell55/taglib-sharp/commits/Net6-update

**Reply by DanCadwellcs** [reply]
I am trying to do Databinding with taglib I need to change any string returns from null to string.Empty.  Also I will to remove all of the null errors found with VS2022.  I will start this tomorrow.  Please let me know if this causes any problems.

The current lib will not work with {bind } at all.
Dan
